geometry_msgs/Transform t
float64 posture
float32 posture_gain
